src/installer/managed/Microsoft.NET.HostModel/ResourceUpdater.cs

```diff
@@ -152,7 +152,7 @@ public enum LoadLibraryFlags : uint
         /// </summary>
         private class SafeUpdateHandle : SafeHandle
```

Comment:
Not your change, but do we also recommend to make concrete SafeHandles `sealed`?

Reply:
Yes.